### ISSUE TITLE
esp smartconfig provisioning fails on ios

### ISSUE THREAD

**Issue by belal410** [abobija/esp-smartconfig-dart#13]
The plugin is working fine with android but trying it with ios returns error message "Socket error: SocketException: Send failed(OS Error : No route to host, errno =65), address = 0.0.0.0, port = 18266"

**Comment by abobija** [comment]
Check if readme section [Breaking Change iOS 14.6/16](https://github.com/abobija/esp-smartconfig-dart#breaking-change-ios-14616) can help with your issue.

**Comment by belal410** [comment]
Yes I have added the multicast network entitlement and still facing the same issue, I guess the issue that the process is still running on port 18266 although the provisioning method is done and the stream is closed; so trying to call the provisioning start method again throws this exception.

**Comment by lucasromeiroTF** [comment]
hello, any evolution with ios16?

Thanks!!!
